README.md

```diff
@@ -126,8 +126,28 @@ jobs:
           fetch-depth: 0
       - uses: xenitab/gitops-promotion@v0.1.0
         with:
-          action: status
           token: ${{ secrets.GITHUB_TOKEN }}
+          action: status
+```
+
+### GitHub App authentication
+
+For simplicity, the above example use a Personal Access Token for authentication. However, in a production setup you probably want to use a [GitHub App](https://docs.github.com/en/developers/apps/building-github-apps/creating-a-github-app). Once you have set up a GitHub app, you can use the [tibdex/github-app-token](https://github.com/tibdex/github-app-token) action to generate a token for the app to access the repository. (In the case of the `build-app` job above, you also want to add `repository: ${{ github.repository_owner }}/my-gitops` since the token should be valid for the repository we dispatch to.)
```

Comment:
Nit: grammar
```suggestion
For simplicity, the above example uses a Personal Access Token for authentication. However, in a production setup you probably want to use a [GitHub App](https://docs.github.com/en/developers/apps/building-github-apps/creating-a-github-app). Once you have set up a GitHub app, you can use the [tibdex/github-app-token](https://github.com/tibdex/github-app-token) action to generate a token for the app to access the repository. (In the case of the `build-app` job above, you also want to add `repository: ${{ github.repository_owner }}/my-gitops` since the token should be valid for the repository we dispatch to.)
```

Reply:
Fixed.